src/crons/EpicGames.ts

```diff
@@ -196,9 +196,9 @@ async function getOfferedGames(
       link: `https://www.epicgames.com/store/fr/p/${
         slugSlashIndex === -1 ? slug : slug.slice(0, slugSlashIndex)
       }`,
-      thumbnail:
-        game.keyImages.find((image) => image.type === 'Thumbnail')?.url ||
-        game.keyImages[0].url,
+      thumbnail: encodeURI(
+        game.keyImages.find((image) => image.type === 'Thumbnail').url,
```

Comment:
I think we should keep the fallback? or throw if there is no thumbnail

Reply:
I think that the Thumbail will always be there. And at worst, it will end up in unhandled rejection, and will be the equivalent of a throw

Reply:
ok, but you need to fix the TypeScript issue somehow